### ISSUE TITLE
[Feature Request] Daily/Frequent Builds - Preferably on Flatpak

### ISSUE THREAD

**Issue by alex285** [olive-editor/olive#1029]
Since that's my first issue here (that is not actually an issue) I want to give many congratulations to Olive developers for this awesome program, that I use everyday for a couple of months now!!

I'd like to request if we can get more recent builds of the editor, preferably on [Flatpak Beta channel](https://github.com/flathub/org.olivevideoeditor.Olive/tree/beta) so we can open some bugs on master

Also I know already there is https://github.com/olive-editor/olive/issues/159 but it would greatly help if the Flatpak build files were upstream on Olive repo, for more attention and discover-ability, plus it makes it easier for people to build and try it from sources 

Thank you!! (cc @eszlari )

**Comment by Simran-B** [comment]
Olive is being rewritten at the moment and therefore not functional. It may not even compile. Therefore, don't expect any new alpha releases in the next couple of weeks.

**Comment by eszlari** [comment]
I could update the beta branch on Flathub, if the cmake install target is fixed, see issue: https://github.com/olive-editor/olive/issues/1008

**Comment by eszlari** [comment]
I updated the Flathub beta branch:
https://github.com/flathub/org.olivevideoeditor.Olive/commit/6bee102c15d04b1e16023e9b308a609b1fe77b97

To install:
```
$ flatpak remote-add flathub-beta https://flathub.org/beta-repo/flathub-beta.flatpakrepo
$ flatpak install flathub-beta org.olivevideoeditor.Olive
$ flatpak run org.olivevideoeditor.Olive//beta
```

**Comment by alex285** [comment]
@eszlari thank you! it doesnt really work, but i will build locally periodically, and PR to beta (properly this time!)

**Comment by shibotto** [comment]
Please when it's ready also keep providing AppImages as a universal cross-distro distribution model.

**Comment by RafaelLinux** [comment]
I periodically look for new AppImage versions, but since may have not been new updates.

**Comment by itsmattkc** [comment]
AppImages, etc. will definitely return in the near future. @eszlari has solved this for time being (I've also fixed #1008 in master so the flatpak won't need to patch it anymore)
